### DISCUSSION TITLE
Different menu per portal

### DISCUSSION THREAD

**Discussion by Tychodewaard** [vanjarosoftware/Vanjaro.Platform#1033] [Q&A] [answered]
Is it possible to have a standard menu on portal 1 and another menu on portal 2?
If so, how? Because currently the menu is in the basic folder that resides on the portals/default.

**Comment by meetmandeep** [comment, marked as answer]
Yes. You can have a different menu on different portals. You can even have different menu on different pages within same portal; or even two different menus on a same page. 

From the Menu Block Settings, there is an option to choose the Menu Template. You can customize this for all blocks on the site or a specific block only.

**Reply by Tychodewaard** [reply]
If I add another menu in /Portals/_default/vThemes/Basic/blocks/Menu/Templates I can select that menu in the menu settings. But...If I select 'Site settings' it also affects other sites within the host. If I select 'Customize this block only' (it is a global block), it doesn't seem to do anything. 

Am I doing something wrong or is it a bug?

**Reply by mksmanish79** [reply]
We are able to duplicate the first issue and it will be fixed in an upcoming release. We are not able to duplicate the global block issue. Can you provide us the steps to duplicate this?
